catalogue/webapp/pages/workv2.js

```diff
@@ -116,7 +116,7 @@ export const WorkPage = ({
                   }
 
                   {work.physicalDescription &&
-                    <MetaUnit headingText='Physical description' text={[work.physicalDescription]} />
+                    <MetaUnit headingText='Physical description' text={[`${work.physicalDescription} ${work.extent} ${work.dimensions}`]} />
```

Comment:
Do we need the spaces given that they give us quite a lot of spaces?
Not that double spaces matter in HTML land.

Reply:
tried it without first, thought it looked better with

Reply:
P.s. #3721 :trollface: 

Although it is a change made without any user input, bar the question around whether people understand `extent`.

And my question now would be why aren't we doing this with other fields that are Marc subtyped fields?

These are just questions around process, not questioning the work at all.

Ping @Heesoomoon